lib/console.js

```diff
@@ -3,12 +3,9 @@
 const util = require('util');
 
 function Console(stdout, stderr) {
-  if (!(this instanceof Console)) {
-    return new Console(stdout, stderr);
-  }
-  if (!stdout || typeof stdout.write !== 'function') {
-    throw new TypeError('Console expects a writable stream instance');
-  }
+  if (!(this instanceof Console)) return new Console(stdout, stderr);
```

Comment:
@cjihrig What style guides do you using? Should I revert all the style changes or only this one?